subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/bintray/ShipkitBintrayPlugin.java

```diff
@@ -55,6 +57,17 @@ public void apply(final Project project) {
             LOG.lifecycle(welcomeMessage);
         });
 
+        bintrayUpload.doLast(task -> {
```

Comment:
I suggest to use "doFirst" - it is more standard approach for pre-condition validation.